agent/acs/handler/acs_handler.go

```diff
@@ -335,7 +335,6 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 
 	client.AddRequestHandler(payloadHandler.handlerFunc())
 
-	// Add HeartbeatHandler to acknowledge ACS heartbeats
```

Comment:
I think this comment is still relevant, no?
We should end up with a couple of heartbeat handlers:
one to ack ACS heartbeats and one to trigger the healthchecks (the latter handler is still TODO)

Reply:
Are we not already acking ACS heartbeats here - https://github.com/aws/amazon-ecs-agent/blob/master/agent/acs/handler/heartbeat_handler.go#L86 🤔

Reply:
Currently the handler exists to ack ACS heartbeats.  
We have a TODO to separate the ACS ack handler from the healthcheck handler (which is also triggered by ACS heartbeat messages)